hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,72 @@
+  emitting a warning when the default (200 millisecond per test case) deadline is exceeded.
+- The ``timeout`` setting has been removed; use :obj:`~hypothesis.settings.deadline`.
+- The ``database_file`` setting has been removed; use :obj:`~hypothesis.settings.database`.
+- The ``perform_health_check`` setting has been removed; use
+  :obj:`~hypothesis.settings.suppress_health_check`.
+- The ``max_shrinks`` setting has been removed; use :obj:`~hypothesis.settings.phases`
+  to disable shrinking.
+- The ``min_satisfying_examples``, ``max_iterations``, ``strict``, ``use_coverage``,
+  and ``max_shrinks`` settings have been removed without user-configurable replacements.
+
+Strategies
+~~~~~~~~~~
+- The ``elements=`` argument is now required for collection strategies.
+- The ``average_size=`` argument was a no-op and has been removed.
+- Date and time strategies now only accept ``min_value`` and ``max_value`` for bounds.
+- :func:`~hypothesis.strategies.builds` now requires that the thing to build is
+  passed as the first positional argument.
+- Alphabet validation for :func:`~hypothesis.strategies.text` raises errors, not warnings,
+  as does category validation for :func:`~hypothesis.strategies.characters`.
+- The ``choices()`` strategy has been removed.  Instead, you can use
+  :func:`~hypothesis.strategies.data` with :func:`~hypothesis.strategies.sampled_from`,
+  so ``choice(elements)`` becomes ``data.draw(sampled_from(elements))``.
+- The ``streaming()`` strategy has been removed.  Instead, you can use
+  :func:`~hypothesis.strategies.data` and replace iterating over the stream with
+  ``data.draw()`` calls.
+- :func:`~hypothesis.strategies.sampled_from` and :func:`~hypothesis.strategies.permutations`
+  raise errors instead of warnings if passed a collection that is not a sequence.
+
+Miscellaneous
+~~~~~~~~~~~~~
+- Applying :func:`@given <hypothesis.given>` to a test function multiple times
+  was really inefficient, and now it's also an error.
+- Using the ``.example()`` method of a strategy (intended for interactive
+  exploration) within another strategy or a test function always weakened
+  data generation and broke shrinking... and now it's an error too.
+- The ``HYPOTHESIS_DATABASE_FILE`` environment variable was deprecated and is
+  no longer supported, as the ``database_file`` setting has been removed.
+- The ``HYPOTHESIS_VERBOSITY_LEVEL`` environment variable was deprecated
+  and is no longer supported.  You can use the ``--hypothesis-verbosity``
+  pytest argument instead, or write your own setup code using the settings
+  profile system to replace it.
+- Using :func:`@seed <hypothesis.seed>` or
+  :obj:`derandomize=True <hypothesis.settings.derandomize>` now forces
+  :obj:`database=None <hypothesis.settings.database>` to ensure results
+  are in fact reproducible.  If :obj:`~hypothesis.settings.database` is
+  *not* ``None``, doing so also emits a ``HypothesisWarning``.
```

Comment:
You removed a bunch of exception types in `9485d18`, but haven't listed them by name here.

Reply:
Now in the Misc section.